experimental/dds/tree2/src/test/simple-tree/object.spec.ts

```diff
@@ -197,91 +207,82 @@ function testObjectLike(testCases: TestCase[]) {
 const tcs: TestCase[] = [
 	{
 		schema: (() => {
-			const _ = new SchemaBuilder({ scope: "test" });
-			const $ = _.object("empty", {});
-			return _.intoSchema($);
+			const _ = new SchemaFactory("test");
+			return _.object("empty", {});
 		})(),
 		initialTree: {},
 	},
 	{
 		schema: (() => {
-			const _ = new SchemaBuilder({ scope: "test" });
-			const $ = _.object("primitives", {
-				boolean: leaf.boolean,
-				number: leaf.number,
-				string: leaf.string,
+			const _ = new SchemaFactory("test");
+			return _.object("primitives", {
+				boolean: _.boolean,
+				number: _.number,
+				string: _.string,
 			});
-			return _.intoSchema($);
 		})(),
 		initialTree: {
 			boolean: false,
-			number: NaN,
+			number: Math.E,
 			string: "",
 		},
 	},
 	{
 		schema: (() => {
-			const _ = new SchemaBuilder({ scope: "test" });
-			const $ = _.object("optional", {
-				boolean: _.optional(leaf.boolean),
-				number: _.optional(leaf.number),
-				string: _.optional(leaf.string),
+			const _ = new SchemaFactory("test");
+			return _.object("optional", {
+				boolean: _.optional(_.boolean),
+				number: _.optional(_.number),
+				string: _.optional(_.string),
 			});
-			return _.intoSchema($);
 		})(),
 		initialTree: {},
 	},
 	{
 		schema: (() => {
-			const _ = new SchemaBuilder({ scope: "test" });
-			const $ = _.object("optional (defined)", {
-				boolean: _.optional(leaf.boolean),
-				number: _.optional(leaf.number),
-				string: _.optional(leaf.string),
+			const _ = new SchemaFactory("test");
+			return _.object("optional (defined)", {
+				boolean: _.optional(_.boolean),
+				number: _.optional(_.number),
+				string: _.optional(_.string),
 			});
-			return _.intoSchema($);
 		})(),
 		initialTree: {
 			boolean: true,
-			number: -0,
+			number: 0,
```

Comment:
Intentional change per our JSON interoperability fix, '-0' is normalized to '0' on write.